internal/builtinfunctions/functions.go

```diff
@@ -550,10 +550,11 @@ func getBindConstantsFunction() schema.CallableFunction {
 		schema.NewDisplayValue(
 			schema.PointerTo("Bind Constants"),
 			schema.PointerTo(
-				"Creates a list of objects with ID `CombinedObject`. "+
-					fmt.Sprintf("Each object has two properties `%s` and `%s`.\n", CombinedObjPropertyItemName, CombinedObjPropertyConstantName)+
+				"Returns a list of objects each containing two properties. "+
 					fmt.Sprintf("Param 1: Value(s) to be included in the `%s` field \n", CombinedObjPropertyItemName)+
-					fmt.Sprintf("Param 2: Value(s) to populate the field `%s` with every output item", CombinedObjPropertyConstantName)),
+					fmt.Sprintf("Param 2: Value(s) to populate the field `%s` with every output item\n", CombinedObjPropertyConstantName)+
+					"The output list items will have an Object ID and schema name as documented in "+
+					"https://arcalot.io/arcaflow/workflows/schemas/#generated-combined-schema-names."),
```

Comment:
Although I appreciate offloading the detailed DRY ID construction algorithm reference, I'm not entirely sure how I feel about a CLI dumping a raw URL. A modern terminal emulator should, on the plus side, allow a control/CMD-click or right click to open the URL directly, but it's rather bulky. Maybe that's the lesser of evils, though.

Reply:
Then again ... it seems you need to empty your lint filter anyway... 😆

Reply:
what do you want me to do here?

Reply:
I have yet to connect the dots on this stuff:  I know that this gets rolled up by `docsgen` into a web page, but how else might a user encounter this text?  (What is the command which would cause it to be dumped onto a terminal?)

Reply:
I really don't know. I'd just mark this resolved and not worry about it... however, Webb has somewhat hijacked this conversation, and like him I'm a bit confused on exactly where this text will be visible. There's not actually a cli command that dumps this documentation, is there? I chose to just review it based on the text on its own; but whether the ugliness of the link is relevant, or whether the link is even useful, depends greatly on the context where it's displayed.